developer/src/kmc-ldml/test/test-testdata-e2e.ts

```diff
@@ -14,7 +14,7 @@ describe('testdata-tests', function() {
     const jsonFilename = makePathToFixture('test-fr.json');
 
     // Compile the keyboard
-    const testData = loadTestdata(inputFilename, {debug: true, addCompilerVersion: false});
+    const testData = loadTestdata(inputFilename, compilerTestOptions);
```

Comment:
Is `{debug: true, addCompilerVersion: false}` set somewhere else?

Reply:
Good catch! Probably doesn't matter for this test but I should have ... included it.
```suggestion
    const testData = loadTestdata(inputFilename, {...compilerTestOptions, debug: true, addCompilerVersion: false});
```